data/korath/korath ships.txt

```diff
@@ -296,6 +314,154 @@ ship "Korath Raider" "Korath Raider (Crippled)"
 	turret "Korath Warder"
 
 
+ship "Korath Raider Mark XLII"
+	sprite "ship/raiderxlii"
+	thumbnail "thumbnail/raiderxlii"
+	attributes
+		category "Medium Warship"
+		"cost" 22895000
+		"shields" 29500
+		"hull" 11000
+		"required crew" 157
+		"bunks" 250
+		"mass" 952
+		"drag" 15
+		"heat dissipation" .6
+		"heat protection" 0.1
+		"hull protection" 0.1
+		"fuel capacity" 700
+		"cargo space" 163
+		"outfit space" 871
+		"weapon capacity" 377
+		"engine capacity" 212
+		weapon
+			"blast radius" 250
+			"shield damage" 3700
+			"hull damage" 1900
+			"hit force" 5500
+	outfits
+		"Korath Grab-Strike" 4
+		"Korath Banisher"
+		"Korath Warder"
+		"Korath Fire-Lance"
+		"Firelight Missile Bank" 2
+		"Firelight Missile" 36
+		
+		"Triple Plasma Core"
+		"Systems Core (Medium)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Fuel Processor"
+		"Korath Repeater Rifle" 196
+		
+		"Thruster (Planetary Class)"
+		"Thruster (Lunar Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive"
+	
+	engine -20 132
+		over
+	engine 20 132
+		over
+	engine 0 142
+		over
+	"reverse engine" -9 -154
+		zoom 0.7
+	"reverse engine" 9 -154
+		zoom 0.7
+	"steering engine" -89 33.5
+		zoom 0.3
+		angle 126
+		left
+	"steering engine" 89 33.5
+		zoom 0.3
+		angle -126
+		right
+	"steering engine" -82.5 30
+		zoom 0.3
+		angle 126
+		left
+	"steering engine" 82.5 30
+		zoom 0.3
+		angle -126
+		right
+	"steering engine" -69.5 -103.5
+		zoom 0.3
+		angle 38
+		right
+	"steering engine" 69.5 -103.5
+		zoom 0.3
+		angle -38
+		left
+	"steering engine" -76 -106
+		zoom 0.3
+		angle 38
+		right
+	"steering engine" 76 -106
+		zoom 0.3
+		angle -38
+		left
+	gun -45 -150 "Korath Fire-Lance"
+	gun 32.5 -1 "Firelight Missile Bank"
+		angle 60
+	gun -32.5 -1 "Firelight Missile Bank"
+		angle 300
+	turret -10 -143 "Korath Grab-Strike"
+	turret 10 -143 "Korath Grab-Strike"
+	turret -36 -141 "Korath Grab-Strike"
+	turret 36 -141 "Korath Grab-Strike"
+	turret 0 41 "Korath Banisher"
+	turret 0 94 "Korath Warder"
+	bay "Fighter" -80 151 back
+	bay "Fighter" 80 151 back
+	bay "Fighter" 28.5 145 back
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the era of shortages and desperation caused by a life in exile."
+	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
+
+
+ship "Korath Raider Mark XLII" "Korath Raider Mark XLII (Crippler)"
+	sprite "ship/raiderxlii"
+	thumbnail "thumbnail/raiderxlii"
+	add attributes
+		"cargo space" -70
+		"outfit space" 70
```

Comment:
While we have special variants that modify base attributes (like the Barb, or Auxiliary), I don't think it makes sense for this variant to modify the base stats since it's literally the same ship (sprite) as the base model.

Although I'm not a content person, so...

Reply:
https://github.com/endless-sky/endless-sky/pull/6955 If you want to continue the conversation here.